src/Storefront/Resources/app/storefront/src/plugin/google-analytics/events/purchase.event.js

```diff
@@ -13,9 +13,10 @@ export default class PurchaseEvent extends AnalyticsEvent
             return;
         }
 
-        const orderNumberElement = DomAccessHelper.querySelector(document, '.finish-ordernumber');
```

Comment:
why not adding `false` as third parameter? :thinking:

Reply:
To be honest, I find the `DomAccessHelper` an unneeded abstraction, when the element is optional. But I can also adjust it to use the `DomAccessHelper` again.

Reply:
let the frontend expert decided :grin:

Reply:
I agree with @aragon999 for me it's also fine to use the native API. 

We also thought about this already and maybe we can use the native API everywhere in the future for `querySelector` instead of `DomAccess` since it does not bring so much benefit. Some other functions can still be used like `getDataAttribute` where a bit more code is abstracted. But `querySelector` does almost the same besides being able to throw a hard error when the element isn't found.

Reply:
@tobiasberge yes, in particular in my plugins I replaced the occurrences of the `DomAccess.querySelector{,All}` as it makes more problems, then it benefits.

Reply:
sound like another PR then :slightly_smiling_face:

Reply:
Yes this should be a separate thing with proper documentation etc. But you don't need to take care of this, for me it's totally ok if you use the native API here.